metrics/metrics.go

```diff
@@ -0,0 +1,71 @@
+package metrics
```

Comment:
any chance we could move this into `pkg`? I feel that keeping the top-level as simple and clean as possible makes the project more approachable. Also, since we don't have a `main.go` at the top level anymore it's not immediately clear where to start when first starting, and adding another top-level directory, for me, doesn't contribute to the approachability.

Reply:
Sure, np